app/code/Magento/Sales/Model/ExpireQuotesFilterFieldsProvider.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Sales\Model;
+
+/**
+ * Provides expire quotes filter fields.
+ */
+class ExpireQuotesFilterFieldsProvider
```

Comment:
Do you really believe that we need to have a separated class to collect the filters?
Or could we just add filters as array parameter to the  `app/code/Magento/Sales/Model/ResourceModel/Collection/ExpiredQuotesCollection.php` ?
I can not come with real case when we need a new class.